jujugui/static/gui/src/app/store/env/controller-api.js

```diff
@@ -463,7 +463,6 @@ YUI.add('juju-controller-api', function(Y) {
       Return information about Juju models, such as their names, series, and
       provider types, by performing a ModelManager.ModelInfo Juju API request.
 
-      @method modelInfo
```

Comment:
I don't think it matters, but why are we removing the '@method' comment statement?